### PR TITLE
Basic Documentation for Spawner Class

### PR DESCRIPTION
**Pull request by tdug**
## Purpose and Motivation

This adds some basic documentation.

## Types of changes

- Documentation
- Argument name change. This class is likely not used directly anywhere.

## To-do list

- [ ] Code is tested
- [ ] All tests are passing
- [x] Updated documentation
- [ ] This PR is ready for review

### CODE REVIEW

File: SCClassLibrary/Common/Streams/Pspawner.sc [hunk 1]
```diff
@@ -1,8 +1,8 @@
 Spawner : Pattern {
-	var <>genStream;
-	var <>priorityQ;
-	var <>now;
-	var <>event;
+	var genStream;
+	var priorityQ;
+	var now;
+	var event;
```
Comment (tdug) [inline SCClassLibrary/Common/Streams/Pspawner.sc:5]: I did not see any cases of these variables used outside of this class. They may as well be private.

File: SCClassLibrary/Common/Streams/Pspawner.sc [hunk 2]
```diff
@@ -29,8 +29,8 @@ Spawner : Pattern {
 		^stream;
 	}
 
-	seq { | pat |
-		pat.embedInStream(event)
+	seq { | pattern |
```
Comment (tdug) [inline SCClassLibrary/Common/Streams/Pspawner.sc:32]: I changed this argument name to be consistent with the argument name in `par`. I wouldn't normally do this, but this class is undocumented, and it's unlikely anyone used the keyword argument syntax for this method in their existing code.

File: HelpSource/Classes/Pspawner.schelp [hunk 3]
```diff
@@ -34,7 +34,7 @@ Stop all substreams of the Spawner.
 ::
 
 note::
-We should move the documentation of above methods to the link::Classes/Spawner:: helpfile...
```
Comment (tdug) [inline HelpSource/Classes/Pspawner.schelp:37]: This was the inspiration for this MR.

**Comment by dyfer** [comment]
Note to self: make a note of the argument change in the Changelog; me and @joshpar think it's fine.

**Pull request merged**